dev-qt/qtwebengine/qtwebengine-5.15.2_p20211210.ebuild

```diff
@@ -110,29 +110,47 @@ PATCHES=(
 	"${WORKDIR}/${PN}-5.15.2_p20211019-jumbo-build.patch" # bug 813957
 )
 
-pkg_preinst() {
-	elog "This version of Qt WebEngine is based on Chromium version 87.0.4280, with"
-	elog "additional security fixes from newer versions. Extensive as it is, the"
-	elog "list of backports is impossible to evaluate, but always bound to be behind"
-	elog "Chromium's release schedule."
-	elog "In addition, various online services may deny service based on an outdated"
-	elog "user agent version (and/or other checks). Google is already known to do so."
-	elog
-	elog "tldr: Your web browsing experience will be compromised."
-}
+qtwebengine_check-reqs() {
+	[[ ${MERGE_TYPE} != binary ]] || return
 
-src_unpack() {
-	# bug 307861
+	# (check-reqs added for bug #570534)
+	#
+	# Estimate the amount of RAM required
+	# Multiplier is *10 because Bash doesn't do floating point maths.
+	# Let's crudely assume ~2GB per compiler job for GCC.
+	local multiplier=20
+
+	# And call it ~1.5GB for Clang.
+	if tc-is-clang ; then
+		multiplier=15
+	fi
+
+	# bug #307861
 	eshopts_push -s extglob
 	if is-flagq '-g?(gdb)?([1-9])'; then
-		ewarn
-		ewarn "You have enabled debug info (probably have -g or -ggdb in your CFLAGS/CXXFLAGS)."
-		ewarn "You may experience really long compilation times and/or increased memory usage."
-		ewarn "If compilation fails, please try removing -g/-ggdb before reporting a bug."
-		ewarn
+		# Debug symbols require significantly more memory for the compiler
+		# to juggle.
+		multiplier=$((multiplier*2))
 	fi
 	eshopts_pop
 
+	local CHECKREQS_DISK_BUILD="7G"
+	local CHECKREQS_DISK_USR="7G"
+	local CHECKREQS_MEMORY=$(($(makeopts_jobs)*${multiplier}/10))G
```

Comment:
I believe the `${}` is redundant inside `$((...))`

Reply:
I think you're right. Ionen mentioned it [here](https://github.com/gentoo/gentoo/pull/23310#discussion_r769216237). It's not my ebuild so I'm happy to use no ${} if you prefer that.